docs/install.md

```diff
@@ -1,93 +1,58 @@
 <!--
 ---
-linkTitle: "Installation"
+linkTitle: "Setting Up Tekton Triggers"
 weight: 2
 ---
 -->
-# Installing Tekton Triggers
+# Setting Up Tekton Triggers
 
-Use this page to add the component to an existing Kubernetes cluster.
+Follow the steps below to set up an official release of Tekton Triggers on your Kubernetes cluster. 
+If you want to test Triggers from `HEAD`, see the
+[Tekton Developer Guide](https://github.com/tektoncd/triggers/blob/master/DEVELOPMENT.md#install-triggers).
 
-- [Pre-requisites](#pre-requisites)
-- [Versions](#versions)
-- [Installing Tekton Triggers](#installing-tekton-triggers-1)
+## Prerequisites
 
-## Pre-requisites
+You must satisfy the following prerequisties to set up Tekton Triggers:
 
-1.  A Kubernetes cluster version 1.18 or later
+* Your cluster must be running Kubernetes version 1.18 or above.
```

Comment:
```suggestion
* You must have a Kubernetes cluster running version 1.18 or later.
```

I would also be in favor of keeping the `kind` instructions as they match `pipeline` install guides:
https://github.com/tektoncd/pipeline/blob/master/docs/install.md#before-you-begin

Reply:
I want to avoid duplication of instructions wherever possible because they can quickly go out of sync between all their different locations. Plus, they must have Pipelines running before installing Triggers, they can't run Triggers without Pipelines, so either way they must go look at Pipelines installation instructions.

Reply:
Restored the `kind` instructions as per review comments.